src/handlers/Popup/PopupHandler.js

```diff
@@ -3,14 +3,17 @@ import { EventEmitter } from 'events'
 
 import config from '../../config'
 import { FEATURES_DEFAULT_POPUP_WINDOW } from '../../utils/enums'
+import { getIFrameOriginObject } from '../../utils/utils'
 import StreamWindow from './StreamWindow'
 
 class PopupHandler extends EventEmitter {
   constructor({ url, target, features, preopenInstanceId }) {
     super()
     const localUrl = url instanceof URL ? url : new URL(url)
-    if (localUrl.hash) localUrl.hash += `&isCustomLogin=${config.isCustomLogin}`
-    else localUrl.hash = `#isCustomLogin=${config.isCustomLogin}`
+    const iframeOrigin = getIFrameOriginObject()
+
+    if (localUrl.hash) localUrl.hash += `&isCustomLogin=${config.isCustomLogin}&namespace=${iframeOrigin.hostname}`
```

Comment:
Move this to a function and use search Params addition

Reply:
done